src/HW_models/NRF_RTC.c

```diff
@@ -332,8 +355,18 @@ void nrf_rtc_regw_sideeffect_TASKS_CLEAR(int i) {
 void nrf_rtc_regw_sideeffect_INTENSET(int i) {
   NRF_RTC_Type *RTC_regs = &NRF_RTC_regs[i];
   if ( RTC_regs->INTENSET ){
+    uint32_t intenset = RTC_regs->INTENSET;
```

Comment:
Should this check only for the bits that were just enabled? (to avoid retriggering an interrupt if it was already set before?)